src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/ProtectedDataCookieTransform.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Cryptography;
+using System.Text;
+using System;
+
+namespace CoreWCF.IdentityModel
+{
+
+    /// <summary>
+    /// Provides cookie integrity and confidentiality using <see cref="ProtectedData"/>.
+    /// </summary>
+    /// <remarks>
+    /// Due to the nature of <see cref="ProtectedData"/>, cookies
+    /// which use this tranform can only be read by the same machine 
+    /// which wrote them. As such, this transform is not appropriate
+    /// for use in applications that run on a web server farm.
+    /// </remarks>
+    public sealed class ProtectedDataCookieTransform : CookieTransform
+    {
+        private const string entropyString = "CoreWCF.IdentityModel.ProtectedDataCookieTransform";
+        private byte[] entropy;
+
+        /// <summary>
+        /// Creates a new instance of <see cref="ProtectedDataCookieTransform"/>.
+        /// </summary>
+        public ProtectedDataCookieTransform()
+        {
+            entropy = Encoding.UTF8.GetBytes( entropyString );
+        }
```

Comment:
Lots of weird formatting in this file. Extra spaces in arguments.

Reply:
Fixed => but you should look at https://referencesource.microsoft.com/#System.IdentityModel/System/IdentityModel/ProtectedDataCookieTransform.cs,e4dd62bad5e9e7bc :D :D ...